src/components/SelectionList/RadioListItem.js

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
 import {View} from 'react-native';
 import Text from '@components/Text';
+import Tooltip from '@components/Tooltip';
 import useThemeStyles from '@styles/useThemeStyles';
 import {radioListItemPropTypes} from './selectionListPropTypes';
 
-function RadioListItem({item, isFocused = false}) {
+function RadioListItem({item, isFocused = false, showTooltip}) {
     const styles = useThemeStyles();
     return (
         <View style={[styles.flex1, styles.alignItemsStart]}>
-            <Text style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}>{item.text}</Text>
+            <Tooltip
+                shouldRender={showTooltip}
+                text={item.text}
+            >
+                <Text
+                    style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold, styles.pre]}
```

Comment:
The styles here and in `UserListItem` are mostly the same. Let's move those styles to `BaseListItem` and pass them as props: `textStyles` `alternateTextStyles`